test/EndToEnd/EndToEnd.Tests.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I imagine you have cleaned up the commented out code from this project in subsequent commits. If so, just ignore this comment.